From 309f3edac3b7d6b8e48c533d5b6837478bad8ad5 Mon Sep 17 00:00:00 2001
From: An Huynh <an.huynh.uj@renesas.com>
Date: Sat, 3 Nov 2018 21:29:37 +0700
Subject: [PATCH 252/909] phy: rcar-gen3-usb2: Add dedicated pin to E3

commit f2d95125e9b042d45eac818edfd6f2ce5d89780f from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds parameter for driver to recognize E3 as device
which have dedicated pins for OTG.

Signed-off-by: An Huynh <an.huynh.uj@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/phy/renesas/phy-rcar-gen3-usb2.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/phy/renesas/phy-rcar-gen3-usb2.c b/drivers/phy/renesas/phy-rcar-gen3-usb2.c
index 1206824158f8..b4e5c7e144ca 100644
--- a/drivers/phy/renesas/phy-rcar-gen3-usb2.c
+++ b/drivers/phy/renesas/phy-rcar-gen3-usb2.c
@@ -410,6 +410,7 @@ static const struct of_device_id rcar_gen3_phy_usb2_match_table[] = {
 	},
 	{
 		.compatible = "renesas,usb2-phy-r8a77990",
+		.data = (void *)RCAR_GEN3_PHY_HAS_DEDICATED_PINS,
 	},
 	{
 		.compatible = "renesas,rcar-gen3-usb2-phy",
-- 
2.17.1

